SpatialGDK/Source/Private/Interop/SpatialSender.cpp

```diff
@@ -441,7 +450,10 @@ void USpatialSender::QueueOutgoingUpdate(USpatialActorChannel* DependentChannel,
 	*Unresolved = UnresolvedObjects;
 
 	FHandleToUnresolved& HandleToUnresolved = PropertyToUnresolved.FindOrAdd(ChannelObjectPair);
-	check(!HandleToUnresolved.Find(Handle));
+	if (HandleToUnresolved.Find(Handle))
```

Comment:
Can you elaborate on what has changed here so that we don't need to assert?

Also, why do you not simply skip rather than removing and adding?

Reply:
Hmm I guess the unresolved UObject pointer can change before we resolve.

Reply:
Ah this is because when we try to create an entity again (because of a failure) we need to remove stuff that we've already queued.